grr/proto/grr_response_proto/api/metadata.proto

```diff
@@ -14,5 +14,5 @@ message ApiGetGrrVersionResult {
 
 // A message type representing the OpenAPI description of the entire API.
 message ApiGetOpenApiDescriptionResult {
-  optional string placeholder = 1;
+  optional string openapi_description = 1;
```

Comment:
Thank you! I updated the use of "openapi" to "open_api" in this PR and kept "description" as we've discussed.

Reply:
I have to slightly disagree here. While it is not a strong preference, I would pick `openai`/`Openapi` variants over `open_api`/`OpenApi`. The reason is that it is kind of one word, not `Open` and `API` separated by space. While not directly related, because different languages have different rules, it also looks like a general convention:

  * [here](https://metacpan.org/pod/Mojolicious::Plugin::OpenAPI) they use `openapi` as the variable/key
  * [here](https://loopback.io/doc/en/lb4/apidocs.openapi-v3.html) they use `openapi-v3` as the package name

Some other precedents with the `Open*` naming:

  * [OpenML](https://www.openml.org/) uses [`openml`](https://github.com/openml) as the name of the GitHub organisation and `openml-*` for language-specific libraries (e.g. [`openml-python`](https://github.com/openml/openml-python))
  * [OpenAI](https://openai.com/) uses [`openai`](https://github.com/openai) as the name of the GitHub organisation and as a Python [PIP package](https://pypi.org/project/openai/)
  * Valve uses [`openvr`](https://github.com/ValveSoftware/openvr) for the name of OpenVR repository

Finally, `openapi` has much better googlability then `open_api` (and searching `open_api` auto-corrects to `openapi`).

Reply:
I believe this is a good point. I personally like `openapi/OpenApi` because "OpenAPI" is, as @panhania  mentioned, sort of a single word with 2 components merged together to form a name and the 2 capital letters point out the 2 components while not making the word awkward to place next to and delimitate from other words (which would be the case with `OpenAPI`, ex: `def OpenAPIImplementation:`). On the other hand, I understand the readability value of keeping a naming scheme convention across a large code base, so I believe the current version of `open_api/OpenApi` is also a good choice.

Please let me know if you (@mbushkov @panhania) would like me to change the current names.

Reply:
@panhania, @alexandrucosminmihai  - thanks for the detailed argument in favour of the `openapi/OpenApi` - agreed!